lib/bolt/cli.rb

```diff
@@ -385,14 +385,16 @@ def outputter
     end
 
     def bundled_content
-      default_content = Bolt::PAL.new([], nil)
-      plans = default_content.list_plans.each_with_object([]) do |iter, col|
-        col << iter&.first
-      end
-      tasks = default_content.list_tasks.each_with_object([]) do |iter, col|
-        col << iter&.first
+      if %w[plan task].include?(options[:subcommand])
```

Comment:
I probably would've implemented this slightly differently (accessing options from random methods feels weird to me, so we could've just skipped calling bundled_content in the first place unless a plan or task). But it works and makes sense.